rfd/0098-securing-docker-images.md

```diff
@@ -0,0 +1,181 @@
+---
+authors: Trent Clarke (trent@goteleport.com)
+state: draft
+---
+
+# RFD 0098 - Delivering secure Teleport OCI Images
+
+# Required Approvers
+* Engineering: @r0mant
+* Security: @reed
+* Product: (@xinding33 || @klizhentas)
+
+## What
+
+This RFD discusses structures and processes to increase the security of the OCI 
+images we use to deliver Teleport to our customers.
+
+## Why
+
+One of our shipping channels is a Docker image. As the provider of that docker
```

Comment:
Well, technically the Docker image is the _artifact_, and ECR is the shipping channel :-)